src/dbus/ofono_handsfree.xml

```diff
@@ -14,7 +14,7 @@
             <arg type="s"/>
             <arg type="v"/>
         </signal>
-        <method name="RequestPhoneNumber"/>
+        <method name="RequestPhoneNumber">
```

Comment:
In case you want to do some cleanup in the file while modifying it you could fix the indentation of this and the signal lines above. Rest of the file has tab indentation while these couple of lines use spaces.

Reply:
I haven't planed to fix indentation. I have executed `xmllint -format` to reformat it in predictable way.

Reply:
Format isn't important for me at all. Valid or invalid xml is.